server/server.go

```diff
@@ -162,7 +165,9 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.LogSourceIPs, "server.log-source-ips-enabled", false, "Optionally log the source IPs.")
 	f.StringVar(&cfg.LogSourceIPsHeader, "server.log-source-ips-header", "", "Header field storing the source IPs. Only used if server.log-source-ips-enabled is true. If not set the default Forwarded, X-Real-IP and X-Forwarded-For headers are used")
 	f.StringVar(&cfg.LogSourceIPsRegex, "server.log-source-ips-regex", "", "Regex for matching the source IPs. Only used if server.log-source-ips-enabled is true. If not set the default Forwarded, X-Real-IP and X-Forwarded-For headers are used")
+	f.BoolVar(&cfg.LogRequestHeaders, "server.log-request-headers-enabled", false, "Optionally log request headers.")
 	f.BoolVar(&cfg.LogRequestAtInfoLevel, "server.log-request-at-info-level-enabled", false, "Optionally log requests at info level instead of debug level.")
+	f.StringVar(&cfg.LogRequestExcludeHeadersList, "server.log-request-exclude-headers-list", "", "Comma separated list of headers to be excluded from the request log. By default headers excluded are Authorization, Cookie and X-Csrf-Token. Other headers specified here will just add to the list.")
```

Comment:
As above, I don't think we want every program using this package to add this verbiage.